pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -1303,7 +1303,7 @@ func TestGlobalResyncOnUpdateAutoscalerConfigMap(t *testing.T) {
 	rev := newTestRevision(testNamespace, testRevision)
 	newDeployment(ctx, t, fakedynamicclient.Get(ctx), testRevision+"-deployment", 3)
 
-	kpa := revisionresources.MakePA(rev)
+	kpa := revisionresources.MakePA(rev, nil)
```

Comment:
Should not we pass the deployment above instead of nil?

Reply:
These are really fixtures - and passing in a deployment doesn't change the fixture so I didn't think it was necessary.